syntax = "proto3";

package client;

message ClNewGame {
  int32 id = 1;
  string fen = 2;
}